models/database.go

```diff
@@ -517,6 +517,27 @@ var databaseSchema = [][]string{
 		`ALTER TABLE artifacts ADD FOREIGN KEY (location_id) REFERENCES backup_locations (id)`,
 		`ALTER TABLE artifacts DROP CONSTRAINT artifacts_service_id_check`,
 	},
+	39: {
+		`CREATE TABLE scheduled_tasks (
+			id VARCHAR NOT NULL,
+			cron_expression VARCHAR NOT NULL CHECK (cron_expression <> ''),
+			type VARCHAR NOT NULL CHECK (type <> ''),
+			start_at TIMESTAMP,
+			last_run TIMESTAMP,
+			next_run TIMESTAMP,
+			data JSONB,
+			disabled BOOLEAN,
+			succeeded INTEGER,
+			failed INTEGER,
```

Comment:
I would suggest to sort fields a different way

Reply:
We don't use it. I just thought It'll be good to have such counters. I can delete it.

Reply:
After one year it can be something like:
```
succeeded: 352
failed: 4
```
I'm not sure that this information can be somehow useful. Probably such counters make sense for a single run with retries, but most likely it will be part of job retries. If there is no need for this now, then let's remove it.